euleros/content.md

```diff
@@ -0,0 +1,35 @@
+%%LOGO%%
+
+# EulerOS
+
+EulerOS provides enhanced security, compatibility and easy-to-use features along with high reliability. It meets the increasing requirements of Linux OS in enterprise applications and provides a compelling choice of open IT platform for users. EulerOS integrates the advanced Linux technologies, delivers more value to enterprise users in terms of high capability, stability, usability and expansibility. And it helps you reallocate resources from maintaining the status quo to tackling new challenges.
+
+## Operating System
+
+Linux/Unix, Other V2.0
+
+## High Lights
+
+-	Extreme Security: EulerOS is one of the most secure operating systems available, which provides variety of security technologies to prevent intrusions and protect your system.
+-	Carrier-class Reliability, Availability and Serviceability: EulerOS customers are able to provide carrier-class ultra-long-term reliability and stability to their users.
+-	High Performance: EulerOS has done a lot of optimization in the compilation, virtual memory, CPU scheduling, IO drive, network and file system and so on.
+
+## License
+
+EulerOS is released under the GPLv2. See the `LICENSE` file in this repository for more information.
+
+## Support Details
+
+Huawei offers 8x5 Comprehensive Support 400 8828 000
+
+## Resources
+
+[Administrators Guide]http://developer.huawei.com/ict/en/doc/site-euleros-administrators-guide-en/index.html/en-us)
+
+## End User License Agreement
+
+By subscribing to this product you agree to terms and conditions outlined in the product End User License Agreement [EULA](http://developer.huawei.com/ict/en/site-euleros/article/privacy-policy)
+
+## Community Support
+
+[zh_CN](http://developer.huawei.com/ict/forum/forum.php?mod=forumdisplay&fid=400257&page=)
```

Comment:
Since this is a community forum, would it make more sense to put this (and possibly the bit above about "Huawei offers 8x5 Comprehensive Support") under "Where to get help?" in the Quick Start section?

(That'd be a `euleros/get-help.md` file to replace the default contents from https://github.com/docker-library/docs/blob/8598ed86200361b68a3e866dddce510a7ea0e8e3/.template-helpers/get-help.md -- [see `xwiki/get-help.md`](https://github.com/docker-library/docs/blob/8598ed86200361b68a3e866dddce510a7ea0e8e3/xwiki/get-help.md) for an example.)

Reply:
done, create a new 'get-help.md' file